book/content/open_research/02/softwarelicenses.md

```diff
@@ -1,5 +1,7 @@
 # Software licenses
 
+*Disclaimer: Good legal advice is timely, specific, and given by an expert; this chapter is none of these. It was written by an engineer, not by a lawyer, and it's a heavily simplified overview of a very complex field. The intent is to give you an overview of the basics, so that you will know when to check whether something you want to do has potential legal ramifications. Don't make any important decisions based solely on the contents of this chapter.*
+
 There are many software licenses in existence. Many of those allow the licensee to do very little, but some give you more freedom to use and re-use the licensed software.
```

Comment:
Can we add a sentence on why do we need licenses as a motivation for this chapter? Or have written that somewhere already?